website/content/commands/partition.mdx

```diff
@@ -95,7 +102,17 @@ $ consul partition create -name "webdev" -description "Partition for admin of we
 
 ### `write`
 
-The `write` subcommand sends a request to the server to create a new admin partition or update an existing partition from its full definition. You can specify an admin partition definition file or use values from `stdin`.
```

Comment:
I moved
> You can specify an admin partition definition file or use values from `stdin`.

to the usage section just below

Reply:
No longer relevant - without the usage subsection, it's no longer necessary to move this part